supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -329,3 +329,12 @@ See xref:release-notes[Release notes] for guidelines about deprecation notices.
 image:images/no.png[no] We will fix this issue in the 18.3.4 release next February.
 
 image:images/yes.png[yes] It is anticipated that an upcoming release will include a fix for this issue.
+
+[[references-to-cost]]
+== References to cost
+
+Avoid all references to the costs and charges of Red Hat products. While the IBM style guide recommends against using the term "free", any references to cost should also be avoided in product documentation since they can confuse users and cause legal concerns. Any cost information is best left to marketing materials.
```

Comment:
The general entry looks really good but I'm adding these small suggested edits to make it more aligned with our IBM and writing style for these items:
- Not to use while to mean although
- Not to use since to mean because
- Avoid 'should',  passives, and be prescriptive
- minimalism -for the last sentence. It's debatable but I think it seems a bit like fluff. And in documentation guidelines maybe it's not applicable to discuss what belongs in marketing material?

```suggestion
Avoid all references to the costs and charges of Red Hat products. Although the _IBM Style Guide_ recommends against using the term "free", also avoid any references to cost in product documentation because they can confuse users and cause legal concerns.
```